### DISCUSSION TITLE
In the program “GnssProcessing”, for parameter: staticPositions, I added the noNetScaleSigma option

### DISCUSSION THREAD

**Discussion by IamLittleMonster** [groops-devs/groops#243] [Show and tell]
In the current version, I found that there is **no separate constraint** for **the scale parameter** when constraining staticPositions. 

Therefore, based on the content on page 143 of [https://openlib.tugraz.at/reprocessing-multiple-gnss-constellations-and-a-global-station-network-from-1994-to-2020-with-the-raw-observation-approach-2022](url), I have added relevant sections. Correspondingly, the **noNetScaleSigma option** has been added to determine the degree of constraint tightness.
![屏幕截图 2024-03-07 084304](https://github.com/groops-devs/groops/assets/51938228/2075a505-2022-44e0-985e-27e02d6a7c7c)

The following is the modified code:
[code.zip](https://github.com/groops-devs/groops/files/14517308/code.zip)

I hope developers can incorporate this change into their next development plan. Additionally, I welcome anyone interested in this area to communicate with me^-^

Cheers,
Hongzhan

**Comment by tmayerguerr** [comment]
Hi Hongzhan,

Thank you very much for your contribution. I have added your source to our main branch. I hope this is in your interest. I have only changed the default value to 0 so that old scripts do not change. 

Best regards
Torsten

**Reply by IamLittleMonster** [reply]
Good job ~
